dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -206,9 +206,41 @@ public User createUser(String userName,
         User user = new User();
         Date now = new Date();
 
-        user.setUserName(userName);
-        user.setUserPassword(EncryptionUtils.getMd5(userPassword));
-        user.setEmail(email);
+        if (StringUtils.isNotEmpty(userName)) {
```

Comment:
Hello @lyleshaw , thx for this improvement. Since you have added several parameters checks, could you please update related unit tests to get those checks covered? Thx